.lintstagedrc

```diff
@@ -1,9 +1,11 @@
 {
-  "*.{ts,tsx,js}": [
-    "eslint --rule 'no-console: error' --fix",
-    "prettier --write src/"
+  "*.ts": [
+    "tsc --noEmit"
```

Comment:
It won't work since when lint-staged is called via husky hooks, `tsc` will omit `tsconfig.json`. 
I believe, we have two options here

- We can go with `bash -c tsc --noEmit` to make it run with the context.
- Or we can use function syntax like this: `() => "tsc --noEmit"`. To make it run, we should change the config file extension to js obviously.

I would prefer the second option here to support cross-platform development. 🤔

Reply:
I didn't check yet but there could be another option. We can specify the config file by using `-p` parameter.